.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,91 @@
+name: 🐛 Bug Report
+description: Report a bug encountered while using Cluster API
+labels: kind/bug
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What steps did you take and what happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please follow the guidelines described in https://github.com/kubernetes-sigs/cluster-api/blob/main/SECURITY_CONTACTS
+      placeholder: "[A clear and concise description on how to REPRODUCE the bug.]"
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: How can we reproduce it (as minimally and precisely as possible)?
+    validations:
+      required: false
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Anything else you would like to add?
+      placeholder: "[Miscellaneous information that will assist in solving the issue.]"
+
+  - type: textarea
+    id: capiVersion
+    attributes:
+      label: Cluster-api version
+      value: |
+        <details>
+
+        </details>
```

Comment:
Q: what is the scope/benefit of using  details here?
Also, ideally, this should be a combo with the list of releases, but I'm not sure we can get this automatically filled (I don't wat to add to the release tasks)

Reply:
I think having a simple field for Cluster API version where folks can enter the version is good enough

Reply:
> Q: what is the scope/benefit of using details here? Also, ideally, this should be a combo with the list of releases, but I'm not sure we can get this automatically filled (I don't wat to add to the release tasks)

This I have copy/pasted from the upstream k8s issue template. Not sure about automatic fill up but there is an option to have a drop down where we could specify list of releases

Reply:
Removed the "<details>"  and made it a simple text field